test/templates/tooltip/component.njk

```diff
@@ -0,0 +1,15 @@
+<!--
+tooltip:
+    properties={},
+    modifier={ block: [] }
+-->
+{% macro tooltip(properties={}, modifier={}, state={}, aria={}) -%}
+<div class="tn-tooltip"{{ aria | aria }}>
+    {%- if modifier.block %}
+    <div class="{{ modifier.block | modifier('tooltip__arrow') }}"></div>
```

Comment:
This should have a `tn-tooltip__arrow` class as well. A modifier class cam only be used with the base class.